specs/AddHostObjectToScriptWithOrigins.md

```diff
@@ -35,7 +35,16 @@ void SubscribeToFrameCreated()
     {
         try
         {
-            List<string> origins = new List<string>(new string[] { "https://appassets.example" });
+            args.Frame.NameChanged += delegate (object sender, CoreWebView2FrameNameChangedEventArgs args)
+            {
+                // Handle frame name changed event
+            };
+            args.Frame.FrameDeleted += delegate (object sender, object args)
+            {
+                // Handle frame deleted event
+            };
+
+            string origins = "https://appassets.example";
```

Comment:
@david-risney  I'm doing something wrong here. Compiler says that `origins` should be string instead of list of strings. Any suggestions? Also where to make a fix to remove the forth parameter from `AddHostObjectToScriptWithOrigins` for .NET?

Reply:
We'll need to modify the .NET code generator. You don't need to do that yet for the API review though. You can just modify the API doc to show the correct API definition and sample code.